Vungle/VungleRouter.m

```diff
@@ -269,6 +268,11 @@ - (void)requestAdWithCustomEventInfo:(NSDictionary *)info
         [self.delegatesDict setObject:delegate forKey:placementId];
     }
     
+    if ([self isAdAvailableForPlacementId:placementId]) {
```

Comment:
Don't we need this check for banner side? `requestBannerAdWithPlacementID`? Just want to make sure ;-)

Reply:
@ashinagawa Hi Aki, thanks a lot for the reminder. In the method `- (void)requestBannerAdWithPlacementID:size:delegate:`, we also have checked the Banner ad is cached: `[self isBannerAdAvailableForPlacementId:placementID size:size]`. 🙂

Reply:
Got it. Thanks for verifying!!